eva/expression/tuple_value_expression.py

```diff
@@ -89,10 +93,12 @@ def signature(self):
             self.col_object, UdfIOCatalogEntry
         ), f"Unsupported type of self.col_object {type(self.col_object)}, expected ColumnCatalogEntry or UdfIOCatalogEntry"
 
+        col_name = self.col_object.name
+        row_id = self.col_object.row_id
         if isinstance(self.col_object, ColumnCatalogEntry):
-            return f"{self.col_object.table_name}.{self.col_object.name}"
+            return f"{self.col_object.table_name}.{col_name}[{row_id}]"
         elif isinstance(self.col_object, UdfIOCatalogEntry):
-            return f"{self.col_object.udf_name}.{self.col_object.name}"
+            return f"{self.col_object.udf_name}.{col_name}[{row_id}]"
```

Comment:
For my education, what is the purpose of `row_id`? Does name already point to a unique entry?

Reply:
It might not in case you drop the table and add a new one with the same name. I wanted to keep name for readability.